job_manager/job_manager.py

```diff
@@ -10,6 +10,44 @@
 
 
 class JobManager:
+    """
+    A class that wraps the RQ worker interface (enqueuement and return retrieval).
+    Does not abstract the choices-to-be-made based on a job's status and availability.
+
+    Attributes
+    ----------
+        _redis : (private) Redis object
+            Manager for Redis in-memory cache that hosts queue for RQ workers
+            and RQ worker results.
+
+        _rq : (private) Queue object
+            Queue onto which jobs are pushed, FIFO, for RQ workers to pick up.
+
+        
+    Methods
+    -------
+        _get_job_hash(func, arglist) (private) :
```

Comment:
wdym when you list things as private?

Reply:
ah, good question. this means that it's not meant to be exposed to the user for use. python doesn't support real private/public interfaces like Java does so the underscore means that I won't necessarily want users touching that part of the method.